apps/common-app/src/examples/RuntimeTests/tests/layoutAnimations/exiting/predefinedExiting.test.tsx

```diff
@@ -0,0 +1,169 @@
+  render,
+  wait,
+  unmockAnimationTimer,
+  clearRenderOutput,
+} from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+import { DurationExitingSnapshots, NoModifierExitingSnapshots, SpringifyExitingSnapshots } from './exiting.snapshot';
+
+const FADE_EXITING = [FadeOut, FadeOutRight, FadeOutLeft, FadeOutUp, FadeOutDown];
+const BOUNCE_EXITING = [BounceOut, BounceOutRight, BounceOutLeft, BounceOutUp, BounceOutDown];
+const FLIP_EXITING = [FlipOutEasyX, FlipOutEasyY, FlipOutXDown, FlipOutXUp, FlipOutYLeft, FlipOutYRight];
+const LIGHTSPEED_EXITING = [LightSpeedOutRight, LightSpeedOutLeft];
+const PINWHEEL_EXITING = [PinwheelOut];
+const ROLL_EXITING = [RollOutRight, RollOutLeft];
+const ROTATE_EXITING = [RotateOutDownLeft, RotateOutDownRight, RotateOutUpLeft, RotateOutUpRight];
+const SLIDE_EXITING = [SlideOutRight, SlideOutLeft, SlideOutUp, SlideOutDown];
+const STRETCH_EXITING = [StretchOutX, StretchOutY];
+const ZOOM_EXITING = [
+  ZoomOut,
+  ZoomOutDown,
+  ZoomOutEasyDown,
+  ZoomOutEasyUp,
+  ZoomOutLeft,
+  ZoomOutRight,
+  ZoomOutRotate,
+  ZoomOutUp,
+];
+
+const EXITING_SETS: Array<[string, unknown[], number]> = [
+  ['LightSpeed', LIGHTSPEED_EXITING, 1600],
+  ['Fade', FADE_EXITING, 1400],
+  ['Bounce', BOUNCE_EXITING, 650],
+  ['Flip', FLIP_EXITING, 1750],
+  ['Pinwheel', PINWHEEL_EXITING, 1000],
+  ['Roll', ROLL_EXITING, 1750],
+  ['Rotate', ROTATE_EXITING, 1600],
+  ['Slide', SLIDE_EXITING, 1800],
+  ['Stretch', STRETCH_EXITING, 1000],
+  ['Zoom', ZOOM_EXITING, 1800],
+];
+
+const ExitingOnMountComponent = ({ exiting }: { exiting: any }) => {
+  const [show, setShow] = useState(true);
+  useEffect(() => {
+    setShow(false);
+  }, []);
+  return (
+    <View style={styles.container}>{show ? <Animated.View exiting={exiting} style={styles.animatedBox} /> : null}</View>
+  );
+};
+
+async function getSnapshotUpdates(exiting: any, waitTime: number, duration: number | undefined, springify = false) {
+  await mockAnimationTimer();
+
+  const updatesContainer = await recordAnimationUpdates();
+  const springExiting = springify ? exiting : exiting.springify();
+  const componentExiting = duration ? springExiting.duration(duration) : springExiting;
+
+  await render(<ExitingOnMountComponent exiting={componentExiting} />);
+
+  await wait(waitTime);
+  const updates = updatesContainer.getUpdates();
+  await unmockAnimationTimer();
+  await clearRenderOutput();
+
+  return updates;
+}
+
+describe.only('Test predefined exiting', () => {
+  describe('exiting on mount, no modifiers', () => {
+    test.each(EXITING_SETS)('Test suite of ${0}Out', async ([_setName, exitingSet, waitTime]) => {
+      for (const exiting of exitingSet) {
+        const snapshotName = (exiting as any).name;
+        const updates = await getSnapshotUpdates(exiting, waitTime, undefined);
+
+        expect(updates).toMatchSnapshots(
+          NoModifierExitingSnapshots[snapshotName as keyof typeof NoModifierExitingSnapshots],
+        );
```

Comment:
Why do we only check for JS updates instead of both JS and native updates? 🤔 The primary goal is to verify that updates applies correctly on the native side as well.